src/decorator-applicator.js

```diff
@@ -1,28 +1,28 @@
 export class DecoratorApplicator {
-  constructor(){
+  constructor() {
     this._first = null;
     this._second = null;
     this._third = null;
     this._rest = null;
   }
 
-  decorator(decorator:Function):DecoratorApplicator{
-    if(this._first === null){
+  decorator(decorator: Function): DecoratorApplicator {
```

Comment:
ESLint can't validate the whitespace around the type annotations, so `decorator(decorator:Function)` won't generate a failure. I added spaces after colons for type annotations because it seems more internally consistent with the rest of the formatting rules.